flask_resources/version.py

```diff
@@ -11,4 +11,4 @@
 and parsed by ``setup.py``.
 """
 
-__version__ = "0.3.0"
+__version__ = "0.4.0"
```

Comment:
```suggestion
__version__ = "0.3.1"
```

This month's release line is out, so a patch to it is good enough.